Functions.Templates/Resources/Resources.resx

```diff
@@ -1945,4 +1945,52 @@ function app.</value>
   <data name="authenticationEvent_trigger_IssuanceStartVer_help" xml:space="preserve">
     <value>API Schema Version for OnTokenIssuanceStart Event</value>
   </data>
+  <data name="sqlIn_displayName" xml:space="preserve">
+    <value>Azure Sql Input Binding</value>
+  </data>
+  <data name="sqlIn_name_label" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="sqlIn_name_help" xml:space="preserve">
+    <value>The name used to identify this binding in your code.</value>
+  </data>
+  <data name="sqlIn_commandText_label" xml:space="preserve">
+    <value>Sql Command Text</value>
+  </data>
+  <data name="sqlIn_commandText_help" xml:space="preserve">
+    <value>The Transact-SQL query command or name of the stored procedure executed by the binding.</value>
+  </data>
+  <data name="sqlIn_commandType_label" xml:space="preserve">
+    <value>Text or StoredProcedure</value>
+  </data>
+  <data name="sqlIn_commandType_help" xml:space="preserve">
+    <value>The Transact-SQL query type which is either Text or StoredProcedure executed by the binding.</value>
+  </data>
+  <data name="sqlIn_parameters_label" xml:space="preserve">
+    <value>Parameters</value>
+  </data>
+  <data name="sqlIn_parameters_help" xml:space="preserve">
+    <value>Parameter values passed to the command during execution as a single string. Must follow the format `@param1=param1,@param2=param2`.</value>
+  </data>
+  <data name="sql_connection_label" xml:space="preserve">
+    <value>Sql Connection string setting</value>
+  </data>
+  <data name="sql_connection_help" xml:space="preserve">
+    <value>The name of the app setting containing your Sql binding connection string.</value>

```

Comment:
@dzsquared should we say "Sql binding" here or "Azure Sql Binding"

Reply:
in descriptions and free text, let's be consistent with "Azure SQL binding"